jcli/src/jcli_lib/address.rs

```diff
@@ -131,7 +131,7 @@ fn mk_delegation(prefix: &str, s: PublicKey<Ed25519>, testing: bool, d: PublicKe
     mk_address_2(prefix, s, d, testing, Kind::Group)
 }
 
-fn mk_account(prefix: &str, s: PublicKey<Ed25519>, testing: bool) {
+pub fn mk_account(prefix: &str, s: PublicKey<Ed25519>, testing: bool) {
```

Comment:
I have to reverse this, since finally I am not really using it outside.

Reply:
yeah, sorry, Sometimes I just make annotations so I do not forget to do so 😅

Reply:
We can make GitHub Actions fail upon meeting TODO/FIXME comments. ;)

Reply:
It would not work. because it was not even noted as a fixme or todo. The thing is that I usually make a self review, if I spot something that shouldn't be there for any reason I make a note in the review so I do not forget to change it 😄